src/applications/enrollment-verification/containers/VerifyEnrollmentsErrorPage.jsx

```diff
@@ -20,7 +23,7 @@ export const VerifyEnrollmentsErrorPage = ({
   useEffect(
     () => {
       if (hasCheckedKeepAlive && !loggedIn) {
-        history.push('/');
+        window.location.href = '/education/verify-school-enrollment';
```

Comment:
I would put this URL in a constants file too.

Reply:
Thanks for the suggestion.